scrapping/ClassType.py

```diff
@@ -0,0 +1,17 @@
+
+class ClassType:
```

Comment:
Asta poate sa extinda enum la fel ca restul de genul ei si nu mai ai nevoie de getClassType.

https://docs.python.org/3/library/enum.html

Reply:
Am vrut sa extind enum, doar ca atunci de ex daca am IE3="abc", si vreau sa scriu in cod Clasa.IE3, aia imi va da valoarea IE3, daca chiar trebuie o sa incerc sa le modific dar imo is ok si asa